src/label/styles/label.css

```diff
@@ -0,0 +1,11 @@
+/* TODO: this should go in a util.css */
```

Comment:
It's not exactly a variable though, and I'm not sure it should be in a theme. I think it and `.hidden` should go in something like `util.css`, and should probably not be themeable. It's hard to think of a case where a widget consumer should be able to override it.